src/Modules/AuthenticationModule/components/ForgetPass/ForgetPass.tsx

```diff
@@ -1,7 +1,164 @@
-import React from 'react'
+import { Box, Button, Grid, TextField, Typography } from "@mui/material";
+import React from "react";
+import forgetImg from "../../../../assets/Images/forget-reset.png";
+import Alert from "@mui/material/Alert";
+import { Link, useNavigate } from "react-router-dom";
+import { useForm, SubmitHandler } from "react-hook-form";
+import axios from "axios";
+import { useToast } from "../../../Context/ToastContext";
+import { getErrorMessage } from "../../../../utils/error";
 
+type Input = {
+  email: string;
+};
 export default function ForgetPass() {
+  const { showToast } = useToast();
+
+  const navigate = useNavigate();
+  let {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<Input>();
+  const onSubmit: SubmitHandler<Input> = async (data) => {
+    try {
+      const response = await axios.post(
+        "https://upskilling-egypt.com:3000/api/v0/admin/users/forgot-password",
```

Comment:
We need to use the base url from a centeralized place to be able to easily update it.